src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -865,17 +865,37 @@ class StoreMonitorCheckInStrategyFactory(ProcessingStrategyFactory[KafkaPayload]
     Does the consumer process unrelated check-ins in parallel?
     """
 
-    def __init__(self, parallel=False) -> None:
-        self.parallel = parallel
+    max_batch_size = 500
```

Comment:
you have two layers of defaults (once in the click comamnd and once here), might get confusing

Reply:
Shall I just make them all required maybe?